libpermian/plugins/anaconda_webui/__init__.py

```diff
@@ -365,7 +363,7 @@ def _start_vm(self):
             location = os_url
 
         # Construct kernel cmdline for --extra-args
-        kernel_cmdline = f'inst.sshd inst.webui inst.webui.remote inst.stage2={os_url}'
+        kernel_cmdline = f'inst.sshd inst.webui inst.webui.remote inst.geoloc=0 inst.stage2={os_url}'
```

Comment:
Hardcoding the inst.geoloc=0 (as workaround) doesn't sound like a good idea to me. I have a feeling it would be better to have this in settings so that this (and possibly other workarounds) can be enabled/disabled by the user without needing to change the code. I'm not calling to have this specific workaround controlled by the settings rather than taking a string from settings that would be put to the kernel cmdline and in this option this workaround could be set.